packages/renderer/src/lib/appearance/appearance-util.ts

```diff
@@ -64,10 +48,10 @@ export class AppearanceUtil {
       return icon;
     }
 
-    const isDark: boolean = await this.isDarkMode();
-    if (isDark && icon.dark) {
+    const isDarkTheme = get(isDark);
```

Comment:
not for this PR, but we need to create a follow-up issue for the getTheme method
it will need to be transformed to a reactive pattern as well.

as here you're using the get method and not the subscribe it means it's only done once.

Reply:
Oh I've probably forgot to change this too

Is there a way/need to unsubscribe in `.ts` file? In `.svelte`, I was using onDestroy, but I think that TS is using a garbage collector, so the unsubscribe method is probably not needed